### PR TITLE
[EDU-1234] Remove old sections and delete older versions of docs content

### PR DESCRIPTION
**Pull request by m-hulbert**
## Description

This PR does the following in order to leave only a single version of each page:

* Deletes the `/versions` directory (v0.8, v1.0 and v1.1) 
* Redirects deleted content to the latest version of a page
* Minor updates to templates and the build process to account for no longer supporting multiple page versions

For sections that have been replaced by the new content structure:

* Deletes the section (including the `/versions` directory)
* Redirects deleted content to the most relevant page in the new content structure
* Updates any links that were pointing to a deleted section 

Commits are structured by section (one for the deletion and redirects, and one for updating any links) to simplify reviewing.

## Review

* Check all deleted content has been redirected

### CODE REVIEW

**Review by aralovelace — APPROVED** [review]
Just checked the src code changes it looks good to me

File: content/partials/types/_rest_client_options.textile [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
+- <span lang="default">queryTime</span><span lang="csharp,go">QueryTime</span><span lang="ruby">:query_time</span><span lang="python">query_time</span> := _false_ If true, the library will query the Ably servers for the current time when "issuing TokenRequests":/auth/token instead of relying on a locally-available time of day. Knowing the time accurately is needed to create valid signed Ably "TokenRequests":/api/rest-sdk/authentication#token-request, so this option is useful for library instances on auth servers where for some reason the server clock cannot be kept synchronized through normal means, such as an "NTP daemon":https://en.wikipedia.org/wiki/Ntpd . The server is queried for the current time once per client library instance (which stores the offset from the local clock), so if using this option you should avoid instancing a new version of the library for each request.<br>__Type: @Boolean@__
```
Comment (Srushtika) [inline content/partials/types/_rest_client_options.textile:1]: Some links to token requests in push docs are going to /auth/token but here I see that you've redirected them to token-request specifically. No links are broken but just raising this as something I noticed.

**Review by Srushtika — APPROVED** [review]
Great work! All good to go. I've tested the redirects and it's all functional except for one-page docs/general/push/activate-subscribe?lang=android. This may not be related to this PR as we discussed but worth fixing. It works for Swift.

Some other unrelated comments:
- content/api/versions/v1.1/token-request-spec.textile — is this file hidden on purpose?
- I noticed a few inconsistencies in URL structures between conceptual and API Ref docs - that's unrelated to this PR but something worth considering in the future. For examples, connect vs connection and statistics vs stats.

File: content/root/platform-customization.textile [hunk 2]
```diff
@@ -66,7 +66,7 @@ h3. Decide whether to use an Ably subdomain, or your own domain
 Ably client libraries connect to two endpoints by default:
 
 - @realtime.ably.io@ - this endpoint is used for all WebSocket and other realtime protocol connections from the Ably client libraries.
-- @rest.ably.io@ - this endpoint is used for all REST-based requests and is optimized for all standard HTTP-based request types including all "REST library requests":/rest, "authentication requests":/general/authentication, and "Comet and JSONP fallback":/general/authentication when WebSockets cannot be used.
+- @rest.ably.io@ - this endpoint is used for all REST-based requests and is optimized for all standard HTTP-based request types including all "REST requests":/api/rest-api, "authentication requests":/general/authentication, and "Comet and JSONP fallback":/general/authentication when WebSockets cannot be used.
```
[the remark this answers is not in the document]
Comment (m-hulbert) [reply content/root/platform-customization.textile]: Fixed in https://github.com/ably/docs/pull/1927/commits/d5d510d2705c279c8434dc9658021872216dd8ca

File: content/partials/types/_rest_client_options.textile (hunk 1 above)
Comment (m-hulbert) [reply content/partials/types/_rest_client_options.textile:1]: Good spot - I don't think this is an issue for now, but I've added it to the push ticket for when that section is reviewed.

**Pull request merged**